client/src/App.js

```diff
@@ -18,7 +19,8 @@ function App() {
           <Login />
         </Route>
         <Route path="/about">
-          <DoctorAbout />
+          {/* <DoctorAbout /> */}
+          <PatientAbout />
         </Route>
```

Comment:
remember to fix it later on

Reply:
I think this will be fixed when we will fetch from the database.